ext/ffi/00_ffi.js

```diff
@@ -326,6 +338,38 @@
             },
           );
         }
+
+        if (needsUnpacking && !isNonBlocking) {
+          const call = this.symbols[symbol];
+          const parameters = symbols[symbol].parameters;
+          const vi = new Int32Array(2);
+          const vui = new Uint32Array(vi.buffer);
```

Comment:
`vi` is only used on the Rust side as a polymorphic `[u8]`? If so, the `Uint32Array` can be allocated instead, and the `vi` may be discarded here, using `vui` in its place.